Source/Core/DolphinQt/NetPlay/NetPlayBrowser.cpp

```diff
@@ -278,6 +271,17 @@ void NetPlayBrowser::OnSelectionChanged()
       ->setEnabled(!m_table_widget->selectedItems().isEmpty());
 }
 
+void NetPlayBrowser::OnUpdateStatusRequested(QString status)
```

Comment:
This should  still be taken by const reference, as `setText` in Qt's API is defined to always take a `const QString&`, so moving in this function wouldn't have any benefit.

Reply:
I don't know why I assumed that `setText(QString&&)` would have been overloaded in the API already.

I'll change it back.

Reply:
Done.

Reply:
(To add to that, I guess Qt will not be able to provide `QLabel::setText(QString&&)` without annoyingly breaking users code. Signal connections that refer to that slot as `&QLabel::setText` would become ambiguous all of the sudden, if the slot was overloaded. Maybe they'll be able to do it in a major version.)